src/status_im/data_store/realm/messages.cljs

```diff
@@ -17,9 +17,7 @@
     (realm/fix-map message :user-statuses :whisper-identity)))
 
 (defn get-by-chat-id
-  "arity-1 returns realm object for queries"
-  ([chat-id]
-   (realm/get-by-field @realm/account-realm :message :chat-id chat-id))
+  "arity-1 returns realm object for queries" 
```

Comment:
i suppose `arity-1` doesn't exist anymore, no?

Reply:
It doesn't so the docstring is wrong now, thanks for pointing it out, will fix it !